src/chrome/content/rules/Switch.ch.xml

```diff
@@ -3,30 +3,232 @@
+		- fl-0-42.unil.cloud.switch.ch
+		- fl-0-44.unil.cloud.switch.ch
+		- fl-2-13.unil.cloud.switch.ch
+		- fl-2-14.unil.cloud.switch.ch
+		- docker.drive.switch.ch
 
-		- mirror	(dropped)
+	Invalid certificate:
+		- kb.pert.switch.ch
+		- aai.switch.ch
+		- metadata.aai.switch.ch
+		- drive-backup.switch.ch (expired)
+		- [a01-a20].drive-backup.switch.ch (expired)
+		- ca.aai.switch.ch
+		- crl.aai.switch.ch
+		- webdav-demo.aai.switch.ch
+		- bresaola.switch.ch
+		- annotations.cast.switch.ch
+		- cast-staging.switch.ch
+		- annotations.cast-test.switch.ch
+		- api.cast-test.switch.ch
+		- keystone.cloud.switch.ch
+		- dashboard.unil.cloud.switch.ch
+		- fl-0-11.unil.cloud.switch.ch
+		- os-0.unil.cloud.switch.ch
+		- comal.switch.ch
+		- tandikat.switch.ch
+		- teramac.switch.ch
+		- teruel.switch.ch
 
+	Too many redirects:
+		- presentation.cast-test.switch.ch
 -->
 <ruleset name="switch.ch">
-
-	<target host="switch.ch" />
+	<target host="switch.ch" />	
+	<target host="www.switch.ch" />
+	<target host="attribute-viewer.aai.switch.ch" />
+	<target host="attribute-viewer-test.aai.switch.ch" />
+	<target host="av.aai.switch.ch" />
+	<target host="idp-monitor.aai.switch.ch" />
+	<target host="ldap-test2.aai.switch.ch" />
+	<target host="rr.aai.switch.ch" />
+	<target host="login.vho.aai.switch.ch" />
+	<target host="tools.vho.aai.switch.ch" />
+	<target host="aai-demo-idp.switch.ch" />
+	<target host="aai-logon.switch.ch" />
+	<target host="aai-viewer.switch.ch" />
+	<target host="x509.aai-logon.switch.ch" />
+	<target host="alphubel.switch.ch" />
+	<target host="bernina.switch.ch" />
+	<target host="bistro.switch.ch" />
+	<target host="aai-demo.switch.ch" />
+	<target host="cast.switch.ch" />
+	<target host="download.cast.switch.ch" />
+	<target host="player.cast.switch.ch" />
+	<target host="producer.cast.switch.ch" />
+	<target host="hslu-ch.producer.cast.switch.ch" />
+	<target host="phlu-ch.producer.cast.switch.ch" />
+	<target host="cast-test.switch.ch" />
+	<target host="download.cast-test.switch.ch" />
+	<target host="player.cast-test.switch.ch" />
+	<target host="producer.cast-test.switch.ch" />
+	<target host="certrepo.switch.ch" />
+	<target host="check.switch.ch" />
+	<target host="dataanalytics.cloud.switch.ch" />
+	<target host="forum.cloud.switch.ch" />
+	<target host="reporting.cloud.switch.ch" />
+	<target host="sandstorm.cloud.switch.ch" />
+	<target host="cylinders.unil.cloud.switch.ch" />
+	<target host="fl-3-137.unil.cloud.switch.ch" />
+	<target host="os.unil.cloud.switch.ch" />
+	<target host="wp7.cloud.switch.ch" />
+	<target host="cylinders.zhdk.cloud.switch.ch" />
+	<target host="fl-0-199.zhdk.cloud.switch.ch" />
+	<target host="fl-0-91.zhdk.cloud.switch.ch" />
+	<target host="fl-4-53.zhdk.cloud.switch.ch" />
+	<target host="fl-6-178.zhdk.cloud.switch.ch" />
+	<target host="os.zhdk.cloud.switch.ch" />
+	<!-- Specific targets can't be used because https://switch-video-production.os.zhdk.cloud.switch.ch/ returns 403 -->
```

Comment:
You can use an exclusion on `^http://switch-video-production\.os\.zhdk\.cloud\.switch\.ch/$` to avoid the implicit test. A custom test url is required anyway.
For wildcards we would require at least three different hosts as test urls.

Reply:
Nice to know! Thank you.